mcu_control/RoverGoal[] goal_list
